README.md

```diff
@@ -73,6 +73,22 @@ $ git clone git@github.com:hackjutsu/Lepton.git
 $ cd Lepton && npm i
 ```
 
+#### On Linux
+
+Currently this library uses `libsecret` so you may need to install it before running `npm install`.
+
+Depending on your distribution, you will need to run the following command:
+
+* Debian/Ubuntu: `sudo apt-get install libsecret-1-dev`
+* Red Hat-based: `sudo yum install libsecret-devel`
+* Arch Linux: `sudo pacman -S libsecret`
+
+Rebuild `node-keytar` (native node module)
+
+```bash
+$ npm run electron-rebuild
+```
+
```

Comment:
Can we avoid treating Linux as a special case?

Reply:
OSes use different things to store secret so `node-keytar` (a native node module) have to deal with this issue.

From what I know, we have to rebuild native node modules b/c some pre-build libs won't work.

Reply:
Thanks for explaining:) I'm hesitant to add extra build steps for particular platforms, especially introducing external dependency by installing prerequisites. This often (significantly) increases the maintenance cost by making it hard to dig out or reproduce bugs, especially when the bug is related to the launch step. Maybe this module is not ready for integration at this moment.

Reply:
Yah, you're right. Using this module can increase failure rate and it's not safe as we want (https://github.com/atom/node-keytar/issues/88)